package.json

```diff
@@ -28,6 +28,7 @@
   "dependencies": {
     "chokidar": "^1.4.3",
     "commander": "^2.9.0",
+    "lodash": "^4.17.2",
```

Comment:
I'd keep the list of dependencies short. Pulling in whole lodash just to have `debounce` method which can be implemented in 3 lines of code seems like a huge overhead to me. Would you be able to refactor this and drop dependency on lodash